PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -2133,7 +2134,7 @@ struct HfCandidateCreatorXic0Omegac0Mc {
         rejectionMask = hfEvSelMc.getHfMcCollisionRejectionMask<BCsInfo, centEstimator>(mcCollision, collSlice, centrality);
       }
       hfEvSelMc.fillHistograms<centEstimator>(mcCollision, rejectionMask);
-      if (rejectionMask != 0) {
+      if (rejectionMask != 0 && !rejectOriginNone) {
```

Comment:
This part of the code is related to the collision selection, while `rejectOriginNone` refers to the candidate orgin. So this looks not correct to me.